homeassistant/components/circuit/__init__.py

```diff
@@ -0,0 +1,34 @@
+"""The Unify Circuit component."""
+
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_NAME, CONF_URL
+from homeassistant.helpers import config_validation as cv, discovery
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = "circuit"
+CONF_WEBHOOK = "webhook"
+
+WEBHOOK_SCHEMA = vol.Schema(
+    {vol.Optional(CONF_NAME): cv.string, vol.Required(CONF_URL): cv.string}
+)
+
+CONFIG_SCHEMA = vol.Schema(
+    {DOMAIN: vol.Schema({vol.Required(CONF_WEBHOOK): [WEBHOOK_SCHEMA]})},
```

Comment:
Please add `cv.ensure_list` to make sure the webhook schema is wrapped in a list.

Reply:
Yes, done.